scp/lib/services/firebase_analytics_service.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:firebase_analytics/firebase_analytics.dart';
+import 'package:firebase_analytics/observer.dart';
+
+class AnalyticsService{
+  final FirebaseAnalytics _analytics = FirebaseAnalytics();
+  
+  FirebaseAnalyticsObserver getAnalyticsObserver() => FirebaseAnalyticsObserver(analytics: _analytics);
+}
```

Comment:
Please make sure that every file ends with a blank line at the bottom. Use the format feature on your IDE/editor, which will format the code for you.
```suggestion
}

```